Dockerfile.base

```diff
@@ -88,12 +90,6 @@ RUN apt update \
 		libxtst6 \
 		openssl \
 		xauth \
-		xdg-utils \
-		xfonts-100dpi \
-		xfonts-75dpi \
-		xfonts-base \
-		xfonts-cyrillic \
-		xfonts-scalable \
```

Comment:
I opted to remove these packages as its removal did not appear to impact E2E testing in any way.